llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -7133,6 +7133,17 @@ SDValue SITargetLowering::lowerBUILD_VECTOR(SDValue Op,
 
 bool
 SITargetLowering::isOffsetFoldingLegal(const GlobalAddressSDNode *GA) const {
+  // Subtargets that use ELF REL relocations (instead of RELA) can only store a
```

Comment:
not a subtarget?

Reply:
Can you suggest a better word? `Subtarget->isAmdHsaOS()` suggests that this *is* a property of the subtarget, or something else here is badly named.

Reply:
The subtarget really shouldn't include this. It's the OS, which is top-level module/triple property